opnsense.logstash.conf

```diff
@@ -1,5 +1,6 @@
 input {
   syslog {
+    port => 5140
```

Comment:
Can you comment it out so it is easy to add but by default it listens on the default syslog port?